demo/site/src/common/blocks/AccordionItemBlock.tsx

```diff
@@ -46,7 +46,7 @@ export const AccordionItemBlock = withPreview(
                     </IconWrapper>
                 </TitleWrapper>
                 <ContentWrapper aria-hidden={!isExpanded} $isExpanded={isExpanded}>
-                    <ContentWrapperInner>
+                    <ContentWrapperInner $isExpanded={isExpanded}>
```

Comment:
Shouldn't this be added to ContentWrapper, which has the aria-hidden attribute?

Reply:
I does not matter i guess, it makes the contentwrapper a bit more complex, but i think its ok to gather all logic there.